classify-cluster-priority/README.md

```diff
@@ -0,0 +1,85 @@
+# Classifying clusters based on priority
+
+Contents:
+
+1. User stories
+2. Current practice
+3. Proposal
+4. Considerations and explanations
+5. Possible applications
+
+## User stories
+
+- As a cluster admin, I want to easily communicate the purpose and importance of a cluster to those interactig with it, so that they can act accordingly.
```

Comment:
```suggestion
- As a cluster admin, I want to easily communicate the purpose and importance of a cluster to those interacting with it, so that they can act accordingly.
```